### DISCUSSION TITLE
ChatGPT style search appeared and disappeared?

### DISCUSSION THREAD

**Discussion by perapp** [squidfunk/mkdocs-material#6728] [Q&A] [answered]
Hello!

I remember it as I, a few weeks ago, went to https://squidfunk.github.io/mkdocs-material/ and typed in a question ChatGPT style like "how do I install mkdocs material?" and a nice summary as response. 
Now it's not there. And I can't find any discussion, PR or documentation on AI powered search. Did I dream it? Or was it there and disappeared? 😨

**Comment by alexvoss** [comment, marked as answer]
You did not dream. The chat functionality was a research prototype that aimed to elicit feedback and help with the development. It has since been turned off and will be replaced with a new iteration when the time is ripe. You can read more about it at https://github.com/squidfunk/mkdocs-material/discussions/6240. 

The discussion will make clear why this was not yet a production feature (cough, hallucinations) but the results were very interesting. For example, I found that the bot was able to combine the Material documentation with general knowledge of CSS. Pretty impressive but such results are still a bit hit and miss.

**Reply by perapp** [reply]
Cool! Then I was really impressed also in real life and not just in a dream of mine! 😃  Really cool stuff. Nice work. I'm looking forward for the next step and a production release with it enabled once ready for that!
